### PR TITLE
FISH-788 - docs about directory config

### PR DESCRIPTION
**Pull request by poikilotherm**
Related to payara/payara#5006, internal ticket FISH-788

Feel free to change as necessary.

### CODE REVIEW

**Comment by AlanRoth** [comment]
Thank you for your PR, I will merge this PR once the PR on Payara Community is merged, in case any changes need to be made there - but LGTM so far

File: docs/modules/ROOT/pages/documentation/microprofile/config/directory.adoc
Comment (OndroMih) [inline docs/modules/ROOT/pages/documentation/microprofile/config/directory.adoc:1]: This new page should also be added to the [nav.adoc](https://github.com/payara/Payara-Community-Documentation/blob/master/docs/modules/ROOT/nav.adoc) file, otherwise it won't appear in the page tree in the left sidebar

**Review by OndroMih — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/documentation/microprofile/config/directory.adoc
Comment (poikilotherm) [reply docs/modules/ROOT/pages/documentation/microprofile/config/directory.adoc:1]: Done in 559f7e9. Thanks for pointing out :+1:

**Comment by OndroMih** [comment]
No need to port to Payara Enterprise docs.

**Pull request merged**
